contracts/GateSeal.vy

```diff
@@ -23,8 +23,8 @@
      sealables are pausable contracts that implement `pauseFor(duration)` interface.
 """
 
-# The maximum GateSeal expiry duration is 1 year.
-MAX_EXPIRY_PERIOD_DAYS: constant(uint256) = 365
+# The maximum GateSeal expiry duration is 3 years.
+MAX_EXPIRY_PERIOD_DAYS: constant(uint256) = 365 * 3
```

Comment:
Since the duration can now span over multiple years (including leap years), we can be more precise by making it 365.25. Or maybe not, not too important I guess